sdv/single_table/base.py

```diff
@@ -974,6 +987,17 @@ def sample_from_conditions(self, conditions, max_tries_per_batch=100,
 
         return sampled
 
+    def _validate_known_columns(self, conditions):
+        """Validate the user-passed conditions."""
+        self._validate_conditions_unseen_columns(conditions)
+        if conditions.dropna().empty:
+            self._raise_condition_with_nans()
+        else:
+            warnings.warn(
+                'Missing values are not yet supported. '
+                'Rows with any missing values will not be created.'
+            )
```

Comment:
I think we should also check here to see if there are no missing values and only warn if there are some missing values.

Reply:
Good catch, thank you!